src/components/aktivitetskrav/vurdering/SendForhandsvarselSkjema.tsx

```diff
@@ -75,6 +80,22 @@ export const SendForhandsvarselSkjema = (
     }
   };
 
+  const SendVarselButton = () => {
+    return (
+      <Button loading={sendForhandsvarsel.isLoading} type="submit">
+        {texts.sendVarselButtonText}
+      </Button>
```

Comment:
Tror vi har testet dette før også: Ettersom den er en del av et form, så funker det ikke å flytte `submit`-knappen ut av Form. Hvis man da trykker på enter i formet, så skjer det ikke noe 😕 Mens ønsket oppførsel er jo da at det som er submit-knappen skal kunne plukke opp enter-trykket.

Reply:
2 ganger før faktisk! 🙃 Jeg husket ikke helt hva vi slet med rundt dette. Er du sikker på ta det er enter-trykket som er problemet? Får ikke til å sende inn ved enter trykk i dev heller – uten å ha fokus på knappen🤔

Reply:
Det funker for meg å trykke enter i både akt.krav og dialogmelding-formene når fokuset er på feks en radioknapp lenger opp i skjemaet🤔.

Reply:
Så rart 😅 Endret det uansett tilbake slik at "submit" ligger inne i formen 😊